excercise1/AIRA/libimportbackuporiginal.py

```diff
@@ -0,0 +1,17 @@
+import myMaths
+
+def main(): #refer to line 16 #Ok so we created a a library (libImport), and now we are trying to use a function but it is not being called upon so the first time to computer goes through this code, it ignore the def main(). Ok Now refer to line 16...again...
+    print("We are doing Mathematics")
+    a = int(input("Enter your first number for maths"))
+    b = int(input("Enter your second number for maths"))
+
+    print(myMaths.myAddition(a, b))
+    print(myMaths.mySubtraction(a, b))
+    print(myMaths.myMultiplication(a, b))
+    print(myMaths.myDivision(a, b))
+
+
```

Comment:
remove unnecessary spaces and camelcase please

Reply:
correct